From 07cfd7700e4084b6cc231e6fccf760a0ee3cff66 Mon Sep 17 00:00:00 2001
From: Daniel Kales <11509575+dkales@users.noreply.github.com>
Date: Mon, 12 Feb 2024 14:35:47 +0100
Subject: [PATCH] patch: do not install linker config for zkllvm nix

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index bbe9ae8..7eafcbe 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -141,7 +141,7 @@ configure_file("${CMAKE_CURRENT_LIST_DIR}/cmake/zkllvm.conf.in"
                "${CMAKE_CURRENT_BINARY_DIR}/zkllvm.conf"
                NO_SOURCE_PERMISSIONS
                @ONLY)
-install(FILES "${CMAKE_CURRENT_BINARY_DIR}/zkllvm.conf" DESTINATION /etc/ld.so.conf.d)
+#install(FILES "${CMAKE_CURRENT_BINARY_DIR}/zkllvm.conf" DESTINATION /etc/ld.so.conf.d)
 
 set(CPACK_DEBIAN_PACKAGE_CONTROL_EXTRA "${CMAKE_CURRENT_LIST_DIR}/cmake/triggers")
 set(CPACK_DEBIAN_PACKAGE_CONTROL_STRICT_PERMISSION TRUE)
-- 
2.34.1

